__tests__/single-device-destructive.test.js

```diff
@@ -108,7 +109,24 @@ describe('Single device - destructive', () => {
             done();
         };
 
-        nRFjprog.verify(device.serialNumber, "./__tests__/hex/connectivity_1.1.0_1m_with_s132_3.0.hex", mockProgressCallback, callback);
+        nRFjprog.verify(device.serialNumber, "./__tests__/hex/connectivity_1.1.0_1m_with_s132_3.0.hex", { }, mockProgressCallback, callback);
+    });
+
+    it('programs a hex string', done => {
+        const callback = (err) => {
+            expect(err).toBeUndefined();
+
+            nRFjprog.verify(device.serialNumber, './__tests__/hex/program.hex', { }, verifyCallback);
+        };
+
+        const verifyCallback = (err) => {
+            expect(err).toBeUndefined();
+            done();
+        };
+
+        const filecontent = fs.readFileSync('./__tests__/hex/program.hex').toString('utf-8');
+
+        nRFjprog.program(device.serialNumber, filecontent, { input_format: nrfjprog.INPUT_FORMAT_HEX_STRING }, callback);
```

Comment:
The `input_format` should ideally be renamed to `inputFormat` in order to be consistent with common JavaScript naming conventions. ESLint will complain about it.